src/components/NibrsDonut.js

```diff
@@ -11,6 +11,10 @@ class NibrsDonut extends React.Component {
     this.state = { hover: null }
   }
 
+  onDetailHover = d => {
```

Comment:
instead of adding this, i think we can leverage the `rememberValue` function that's there, and pass that down to the `NibrsDonutDetails` component and then update the `onMouseOver={() => onMouseOver(d.key)}` to just be `onMouseOver={rememberValue(d.key)}`